src/tests/accommodationTest.js

```diff
@@ -0,0 +1,85 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import dotenv from 'dotenv';
+import app from '../index';
+import mockData from './mock/mockData';
+import accommodationMockData from './mock/accommodationMockData';
+
+chai.use(chaiHttp);
+chai.should();
+
+dotenv.config();
+
+let userToken;
+
+describe('Accomodation Ratings Tests', () => {
+  before((done) => {
+    chai.request(app)
+      .post('/api/v1/auth/signin')
+      .send(mockData.user1)
+      .end((err, res) => {
+        userToken = res.body.data;
+        done();
+      });
+  });
+
+  it('should not be able to save a rating if the rating provided doesn\'t follow the correct format', (done) => {
+    chai.request(app)
+      .post('/api/v1/accomodations/ratings')
+      .send(accommodationMockData.wrongRating)
+      .set('token', userToken)
+      .end((err, res) => {
+        res.should.have.status(400);
+        res.body.should.be.an('object');
+        done();
+      });
+  });
+
+  it('should not be able to save a rating if the review provided doesn\'t follow the correct format', (done) => {
+    chai.request(app)
+      .post('/api/v1/accomodations/ratings')
+      .send(accommodationMockData.wrongReview)
+      .set('token', userToken)
+      .end((err, res) => {
+        res.should.have.status(400);
+        res.body.should.be.an('object');
+        done();
+      });
+  });
+
+  it('should not be able to save a rating if accomodationId provided doesn\'t follow the correct format', (done) => {
+    chai.request(app)
+      .post('/api/v1/accomodations/ratings')
+      .send(accommodationMockData.wrongAccommodationId)
+      .set('token', userToken)
+      .end((err, res) => {
+        res.should.have.status(400);
+        res.body.should.be.an('object');
+        done();
+      });
+  });
+
+  it('should be able to save a rating', (done) => {
+    chai.request(app)
+      .post('/api/v1/accomodations/ratings')
+      .send(accommodationMockData.correctRating)
+      .set('token', userToken)
+      .end((err, res) => {
+        res.should.have.status(201);
+        res.body.should.be.an('object');
+        done();
+      });
+  });
+
+  it('should be not be able to save a duplicate rating', (done) => {
+    chai.request(app)
+      .post('/api/v1/accomodations/ratings')
+      .send(accommodationMockData.correctRating)
+      .set('token', userToken)
+      .end((err, res) => {
+        res.should.have.status(409);
+        res.body.should.be.an('object');
```

Comment:
@ivymwende to make the tests strong, It would be nice to check everything, for example `        expect(res.body.message).eql('you are not a travel administrator');
`